routes/socket/user-events.js

```diff
@@ -2537,6 +2537,10 @@ module.exports.handleUpdatedGameSettings = (socket, passport, data) => {
 			const userIdx = userList.findIndex(user => user.userName === passport.user);
 
 			for (const setting in data) {
+				if (setting == 'blacklist') {
+					data[setting].splice(0, data[setting].length - 30);
```

Comment:
If we're on the path of saving memory and being concise this can just be 
`data.blacklist.splice(30);` 😅

Reply:
@Vigasaurus that removes everything after the 30th element, right? What #1230 says is to remove everything before the last 30 elements - say you have a BL Bob John Alice. You want to add Eve, but your BL is full. Your BL would be John Alice Eve afterwards. There should be some kind of warning, maybe?

Reply:
Yeah good call, maybe this needs some extra re-working too, because I meant to a while ago combine #794 , #852 , and #1230 to one issue for revamping the BL system. I think your current implementation is fine for now then - and I'll work on combining those issues later.